public/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+	<!-- This is a loader for JXML Applications / Modules -->
+	<head>
+		<script src="lib/system.js"></script>
+	</head>
+	<body>
+		<script>
+			System.config({ baseURL: 'lib' });
+
+			var params = {
+				app: 'jx/example/KitchenSink',
+				renderer: 'jx/ui/html/HTMLRenderer'
+			};
+
+			location.search.replace(/\??(([^=]*)=)?([^&]+)&?/g, function(_, _, k, v) {
```

Comment:
This regex does not seem to account for trailing `/`.
Should it be something like `/\??(([^=]*)=)?([^&]+)[^/]&?/g`?